domain/secret/service/params.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package service
+
+import (
+	"time"
+
+	"github.com/juju/juju/core/leadership"
+	"github.com/juju/juju/core/permission"
+	"github.com/juju/juju/core/secrets"
+)
+
+// CreateSecretParams are used to create a secret.
+type CreateSecretParams struct {
+	UpdateSecretParams
+	Version int
+
+	// One owner must be non-nil.
```

Comment:
Probably add Validate() for this.

Reply:
That's all todo - this is just the skeleton for now. Only the minimal structs and not implemented service methods are added here.  The purpose of this PR is to start the facade rework. The service implementation comes next and is not done here.